File: php/MASTER_SLAVE_WHOLE_PART_ABSTRACT_FACTORY/datos/DatosCliente.php
<?php

/**
 * Description of DatosCliente
 *
 * @author giltenorio
 */
class DatosCliente {

    public $nombre;
    public $apellidoPaterno;
    public $apellidoMaterno;
    public $tipoCliente;
    public $edad;

    public function getDatosCliente() {
        $informacion = "<h2>Datos de Cliente</h2>";
        $informacion .= "Nombre: " . $this->nombre;
        $informacion .= "<br/>Apellido Paterno: " . $this->apellidoPaterno;
        $informacion .= "<br/>Apellido Materno: " . $this->apellidoMaterno;
        $informacion .= "<br/>Edad: " . $this->edad;
        $informacion .= "<br/>Tipo de cliente: " . $this->tipoCliente . '<br/>';

        return $informacion;
    }

    public function setDatosCliente($nombre, $apellidoP, $apellidoM, $edad, $tipo) {
        $this->nombre = $nombre;
        $this->apellidoPaterno = $apellidoP;
        $this->apellidoMaterno = $apellidoM;
        $this->edad = $edad;
        $this->tipoCliente = $tipo;
    }

}

File: python/BUILDER_COMPOSITE/BUILDER_COMPOSITE/Reservacion/DatosCliente.py
class DatosCliente:
    
    nombres         : str
    apellidoPaterno : str
    apellidoMaterno : str
    edad            : int
    tipoCliente     : str
    
    def setDatosCliente(self, nombres,
                              apellidoPaterno,
                              apellidoMaterno,
                              edad,
                              tipoCliente):
        self.nombres         = nombres
        self.apellidoPaterno = apellidoPaterno
        self.apellidoMaterno = apellidoMaterno
        self.edad            = edad
        self.tipoCliente     = tipoCliente